Apps/Sandcastle/gallery/development/Terrain Performance.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Measure terrain loading and rendering performance.">
+    <meta name="cesium-sandcastle-labels" content="Development">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+var scene = viewer.scene;
+var camera = scene.camera;
+var globe = scene.globe;
+var statistics = Cesium.RequestScheduler.statistics;
+
+// var cesiumTerrainProviderMeshes = new Cesium.CesiumTerrainProvider({
+//     url : 'http://localhost:8001/2018-07-11b',
+//     requestWaterMask : true,
+//     requestVertexNormals : true
+// });
+// viewer.terrainProvider = cesiumTerrainProviderMeshes;
```

Comment:
Is this commented code still needed here anymore? Was this just for testing out the modified terrain/bvh?